frontend/src/components/userPage/UserPageContent.jsx

```diff
@@ -96,6 +100,13 @@ const UserPageContent = ({ user }) => {
         ))
     }
 
+    const fetchUserRideshares = () => {
+        fetch(`${url}/ridesharing/user/${user.username}`)
+            .then((response) => response.json())
+            .then((data) => setUserRideshares(data))
+            .catch((error) => console.error('Error fetching:', error))
+    }
```

Comment:
UserRideshares is not used in any other component. Why not put it in DisplayUserRidesharings? The UserPageContent.jsx looks pretty overloaded.

Reply:
Done